esp/applications/common/configsec.yaml

```diff
@@ -0,0 +1,10 @@
+authNZ:
+  configsec:
+    SecurityManager:
+      type: AccurintSecurity
```

Comment:
This feels like it should be in the LN repo rather than the HPCC one...

Reply:
I can remove it.  It's a working example of you should be able to configure any SecurityManager based plugin now.  I hope to move this particular plugin to open source at some point, but would change the type from "AccurintSecurity" when I did, the name makes no sense anyway.  This plugin is great for testing and could be used for security unit tests as well.